tox.ini

```diff
@@ -48,10 +48,12 @@ commands =
     py.test --create-db -n 2 -m 'not es_tests' -v src/olympia/ --ignore src/olympia/addons/ --ignore src/olympia/devhub/ --ignore src/olympia/editors/ --ignore src/olympia/amo/ --ignore src/olympia/users/ {posargs}
 
 [testenv:ui-tests]
-passenv = DISPLAY
-install_command = pip install {packages}
-deps = -rtests/ui/requirements.txt
-commands = pytest --verify-base-url --driver Firefox tests/ui {posargs}
+recreate = True
```

Comment:
It's good that it reuses the cache, that just makes the installation quicker. Unless it's also caching the `.tox` directory, we still don't need this. Even if we were, Tox should automatically recreate if the dependencies change.